.travis.yml

```diff
@@ -0,0 +1,39 @@
+language: python
+sudo: true
+branches:
+  only:
+    - master
+    - /^\d\.\d+$/
+    - /^\d\.\d+\.\d+(rc\d+|\.dev\d+)?$/
+matrix:
+  include:
+    - python: 3.4
+      env: TOXENV=py34
+    - python: 3.5
+      env: TOXENV=py35
+    - python: 3.6
+      env: TOXENV=py36
+    - python: 3.6
+      env: TOXENV=docs
+before-install:
+  - sudo apt-get -qq update
+  - sudo apt-get install -y libavahi-compat-libdnssd-dev
+install:
+  - pip install -U tox twine wheel codecov
```

Comment:
why twine? This will be needed only when the travis CI is working well and we want to do automatic uploads to pip, correct?

Reply:
I would like this to happen in a separate change - after we see that CI is running OK. Do you agree?

Reply:
yes, I just have it in there so when you are ready, you can just uncomment `deploy` and fill in the details.